infra_monitored_account/app.py

```
@@ -10,25 +11,31 @@
 from lib.settings import Settings
 from lib.settings.cdk import settings_validator
 
-if "STAGE_NAME" in os.environ:
-    pass
-else:
-    exec('raise ValueError("Environment variable STAGE_NAME is undefined")')
-STAGE_NAME = os.environ["STAGE_NAME"]
-
-PROJECT_NAME = "salmon"
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
 
 settings = Settings.from_file_path("../config/settings")
 settings_validator.validate(settings)
 
 app = cdk.App()
+
+STAGE_NAME = app.node.try_get_context("stage-name")
+if STAGE_NAME is None:
+    logging.info('stage-name is not set. Using "default".')
+    STAGE_NAME = "default"
```

I'd rather throw an error instead of setting a default stage_name